physical/raft/bolt_64bit_test.go

```diff
@@ -43,3 +43,31 @@ func Test_BoltOptions(t *testing.T) {
 		})
 	}
 }
+
+// TestMmapFlags tests the getMmapFlags function, ensuring it returns the appropriate integer representing the desired mmap flag.
+func TestMmapFlags(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name               string
+		disableMapPopulate bool
+	}{
+		{"MAP_POPULATE is enabled", false},
+		{"MAP_POPULATE disabled by env var", true},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+
+		t.Run(tc.name, func(t *testing.T) {
+			if tc.disableMapPopulate {
+				os.Setenv("VAULT_RAFT_DISABLE_MAP_POPULATE", "true")
```

Comment:
It might be nice to replace `os.Setenv` with https://pkg.go.dev/testing#T.Setenv which cleans up after itself when the test finishes so that env vars are reset back to what they were before the test ran.